migrations/versions/917195d9b5e9_add_messageembedding_table.py

```
@@ -0,0 +1,117 @@
+        schema=schema,
+    )
+    op.create_index(
+        "idx_message_embeddings_created_at",
+        "message_embeddings",
+        ["created_at"],
+        schema=schema,
+    )
+
+    # Create HNSW index for vector similarity search
+    op.execute(f"""
+        CREATE INDEX idx_message_embeddings_embedding_hnsw 
```

Flake8 (7.2.0)

[warning] trailing whitespace

(W291)